liwords-ui/.eslintrc

```diff
@@ -46,6 +46,13 @@
         "ignoreComments": true
       }
     ],
+    "no-restricted-imports": ["error", {
```

Comment:
We do some necessary initialization of dayjs in the utils module, so we wanna make sure that nobody imports it without that initialization